### ISSUE TITLE
Invalid leading zero

### ISSUE THREAD

**Issue by HomemBravo** [yunojuno/django-package-monitor#10]
First of all, thank you very much for the previous fix 👍 

We I tried with my own project packages there was another error that I would like to share:

ValueError at /package_monitor/reload/
Invalid leading zero in minor: '0.04.17'

The version is related to Unicode==0.04.17

Is it something related to semantic? If so, warn me if I am on the right repo.

Kind regards,

Andre Carpinteiro

**Comment by hugorodgerbrown** [comment]
Good question - the versions are parsed by semver, so it may be outside our
control. I'll check later when I'm back home.

**Comment by hugorodgerbrown** [comment]
@HomemBravo I've taken a look - and yes, it is `semantic-version` that is raising the error. However, I couldn't install or find the package `Unicode` to test with. Is this a real package?

**Comment by HomemBravo** [comment]
Terrible sorry for the confusion:

https://pypi.python.org/pypi/Unidecode

**Comment by hugorodgerbrown** [comment]
The line that blows up is in the semantic_version project, here -
https://github.com/rbarrois/python-semanticversion/blob/master/semantic_version/base.py#L205

```
if _has_leading_zero(minor):
    raise ValueError("Invalid leading zero in minor: %r" % version_string)
```

So it looks like this is explicitly forbidden - which means it's
unlikely to be amended.

The workaround within this project is to handle unparseable versions.

**Comment by hugorodgerbrown** [comment]
@HomemBravo I am closing this issue, and have opened a new issue (#11) around better handling of errors.

**Comment by hugorodgerbrown** [comment]
@HomemBravo this has been fixed (see #12 ) and uploaded to PyPI as v0.3.5
